### DISCUSSION TITLE
My Initial Take on Gnome Software Icon

### DISCUSSION THREAD

**Discussion by ochi12** [ubuntu/yaru#4155] [Ideas]
This concept design is derived from the app-center icon.
I was supposed to put this to my ![YaruX](https://github.com/ochi12/YaruX) repo but I designed it to be a dependency for Yaru so there is no way I can override the default gnome-software icon without manually removing and replacing this icon in my Yaru install. 

One thing I can say is the old gnome-software logo generally look ugly(at least for me). I see it as a briefcase instead of a shopping bag.

I didn't started 16, 24, 32, 48 sizes and symbolic yet since I don't know yet if this will go anywhere

## NEW
![image](https://github.com/user-attachments/assets/6d20c118-30fc-446c-9251-eb388bdedf9e)

## CURRENT
![image](https://github.com/user-attachments/assets/79fb169e-74b2-43de-ac64-57d20da69386)

**Comment by Feichtmeier** [comment]
I think this is a good idea we should replace all old bag shape icons. 

You could create a pull request and we ask @ubuntujaggers for a review

**Reply by ochi12** [reply]
Done. https://github.com/ubuntu/yaru/pull/4157

**Reply by ochi12** [reply]
@Feichtmeier  I wanna start a project of replacing the dark when in darkmode to light. Basically a way is to just add a function "render-cursors.py" to invert the icon colors (white outline to black then black fill to white). 

I'm literally new to github and contributing to open source projects generally. I just wanna ask if there's any workflow so that I can ask if which part of the code should I change, basically which is which. I have the ideas but I'm super weak right now

**Reply by Feichtmeier** [reply]
Cool thanks! We might need to wait for the others to review this
If you have questions we have a discord server
https://discord.gg/kFqNR5T9
